ethportal-api/src/types/consensus/beacon_block.rs

```diff
@@ -276,8 +276,10 @@ mod test {
         .expect("cannot find test asset");
         let value: Value = serde_yaml::from_str(&value).unwrap();
         let content: SignedBeaconBlockDeneb = serde_json::from_value(value.clone()).unwrap();
-        let serialized = serde_json::to_value(content).unwrap();
-        assert_eq!(serialized, value);
+        let serialized = serde_json::to_value(content.clone()).unwrap();
+        let deserialized_content: SignedBeaconBlockDeneb =
```

Comment:
This seems redundant with line 278.

Reply:
Are we removing the assert then?

Reply:
`fix: can't compare Value's now as one has a number the other has type String`